main.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	acorn "github.com/acorn-io/acorn/pkg/cli"
+	acorn "github.com/acorn-io/runtime/pkg/cli"
```

Comment:
```suggestion
	runtime "github.com/acorn-io/runtime/pkg/cli"
```

Reply:
This seems to be referring to the actual `acorn` CLI command, so I would be inclined to keep it as is. Would you still feel better about changing it?

Reply:
good point. I'm fine with it as is.